pkg/restore/mocks/item_action.go

```diff
@@ -1,19 +1,5 @@
-/*
```

Comment:
Please don't remove the license header 😄

Reply:
sorry, it wasn't me - it was mockery ;)